go/base/tests/test_tasks.py

```diff
@@ -0,0 +1,141 @@
+import go.base.utils
+from go.conversation.view_definition import ConversationViewDefinitionBase
+from go.vumitools.conversation.definition import ConversationDefinitionBase
+from go.vumitools.metrics import ConversationMetric
+from go.vumitools.tests.helpers import djangotest_imports
+from go.base.tasks import (
```

Comment:
You'll need to move the `from go.base.tasks` and `import go.base.utils` imports into the `with djangotest_imports` block below so that the `VUMIGO_SKIP_DJANGO` test run doesn't try to import them.